PowerValidatedSolutions.psm1

```diff
@@ -48145,6 +48145,383 @@ Function Test-WMSubnetInput {
 }
 Export-ModuleMember -Function Test-WMSubnetInput
 
+Function Invoke-GeneratePrivateKeyAndCsrFiles {
+    <#
+        .SYNOPSIS
+        Generate Private Key and CSR files using Openssl, common name and neccesary parameters.  
+
+        .DESCRIPTION
+        The Invoke-GeneratePrivateKeyAndCsrFiles cmdlet generate private key and CSR files using Openssl, common name and input parameters.  
```

Comment:
```suggestion
        The Invoke-GeneratePrivateKeyAndCsrFiles cmdlet generates the private key and certificate signing request (CSR) files using Openssl, using a common name and input parameters.  
```

Reply:
Addressed in commit 53039acbaaffafaf219274cac1a3684735130f53